### PR TITLE
Fix homepage RTL

### LINKED ISSUE #765: RTL broken on homepage
## Description

The homepage is using the `blank` layout which doesn't have basic i18n wired up for setting the `dir` attribute.

## Reproduction

1. Set your locale to an RTL language like `ar`
2. Visit the homepage
3. Note that nothing is in RTL

## Expectation

RTL should work on all pages.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Fixes

Fixes #765 by @sarayourfriend

## Description

Fixes the homepage RTL problems.

1. The logo and brand mark were in a single SVG making it impossible for them to re-order based on reading direction. Splitting them into two separate SVGs makes this possible. It does make using them a little bit more of a hassle, but this shouldn't be something we're using in a bespoke manner more than a few times, so I don't see this as a maintainability problem. If these logos or brand marks ever change it'll require careful manual positioning fixes anyway regardless of if it's a single or two SVGs.
2. Some non-RTL compliant positioning classes were being used, so this switches them to use the relative margin/padding classes.

## Testing Instructions

Load up this branch and run `pnpm dev`.

1. Visit `/` and verify that the homepage looks good at all screen widths
2. Visit `/ar` and verify that the homepage looks good at all screen widths and that text is aligned
3. Visit `/404` and verify that the logo and brand mark look good
4. Visit `/ar/404` and verify that the logo and brand mark look good

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/pages/index.vue [hunk 1]
```diff
@@ -6,20 +6,29 @@
     <header
       class="flex-grow w-full lg:w-auto lg:min-w-[32rem] xl:min-w-[64rem] box-border flex flex-col justify-between lg:justify-center"
     >
-      <VLogoButton class="lg:hidden ms-3" :auto-resize-logo="false" />
+      <VLogoButton
```
Comment (obulat) [inline src/pages/index.vue:9]: Neither header nor `VLogoButton` have a color class, so the color of the logo becomes black instead of dark charcoal.

**Comment by obulat** [comment]
Do we really need to change the logo direction? I couldn't find anything specifically on this, but Ahmad Shadeed's RTL-styling blog uses the same logomark for RTL, without changing direction, for Smashing Magazine: https://rtlstyling.com/posts/rtl-styling#3.-differences-in-word-sizes-between-languages .

**Comment by sarayourfriend** [comment]
@obulat I don't think we necessary do, but in the case of the homepage, the LTR design looks very unbalanced and awkward with out a flipped logo mark. There's no sensible way to have the logo mark "hang" off the leading edge of the page like it does on the LTR version.

If @panchovm has any ideas of how to do this without flipping the logo and brand mark, then we can implement that. But as it stands I couldn't come up with anything that looked decent.

**Comment by obulat** [comment]
> @obulat I don't think we necessary do, but in the case of the homepage, the LTR design looks very unbalanced and awkward with out a flipped logo mark. There's no sensible way to have the logo mark "hang" off the leading edge of the page like it does on the LTR version.

Oh, I didn't realize that this is mainly a solution to fix the "hanging" part of the large screen logo, thank you for explaining!

By the way, the color of the logo in the homepage header on screens smaller than `lg` is still black.

**Comment by panchovm** [comment]
I love ❤️ @sarayourfriend's solution of flipping the logo. It embraces the RTL logic as the most prominent use of Openverse will be through the symbol rather than symbol and wordmark. Therefore, switching positions is a great idea.

**Comment by obulat** [comment]
I *can* see the difference in colors. I guess working closely with @panchovm on the header fixes helped me learn to pay more attention to colors and spacings 😀

**Review by obulat — APPROVED** [review]

**Comment by sarayourfriend** [comment]
I envy your eyes then! I looked very hard and close and really couldn't tell the difference 😆 Thanks for lending your eyes 🙏

**Comment by panchovm** [comment]
Although let me say that @obulat's eagle eyes date longer than the header work.

**Comment by zackkrida** [comment]
Monitor color profiles could also play a major role here 😆

**Review by zackkrida — CHANGES_REQUESTED** [review]
This looks great, I do see one visual regression. Probably as a consequence of separating the logo text and icon, the logo text is no longer aligned with the rest of the content:

> after

CleanShot 2022-02-07 at 12 40 37@2x

> before

CleanShot 2022-02-07 at 12 45 42@2x

The simplest fix might be for the logo text to be in the same container as the other content, and for the logo icon to be absolutely positioned relative to that container.

**Comment by sarayourfriend** [comment]
Ah, that's a much wider browser window configuration than I'm able to achieve with my current set up 😅 I definitely ran into similar problems but didn't realize the margins would keep moving as the view got even larger.

I'll try the absolute positioning trick you recomended.

**Comment by sarayourfriend** [comment]
@zackkrida updated, please let me know if this works for you!

**Review by zackkrida — APPROVED** [review]
Looks perfect!

**Pull request merged**